Pulumi.main.yaml

```
@@ -1,5 +1,6 @@
 config:
-  abax-minuba:host: abax-minuba.branches.no
+  abax-minuba:frontend-host: abax-minuba.branches.no
+  abax-minuba:api-host: api.abax-minuba.branches.no
   abax-minuba:tag: main-673a2d7
   abax-minuba:ui-image: europe-north1-docker.pkg.dev/branches-org-main/branches-org-main/abax-minuba/ui
   abax-procore:host: procore-abax.branches.no
```

```suggestion
  abax-minuba:ui-image: europe-north1-docker.pkg.dev/branches-org-main/branches-org-main/abax-minuba/ui
  abax-minuba:server-image: europe-north1-docker.pkg.dev/branches-org-main/branches-org-main/abax-minuba/server
  abax-procore:host: procore-abax.branches.no
```

Has this image been built and published